cli/tests/070_location.ts

```diff
@@ -1,4 +1,5 @@
-// TODO(nayeemrmn): Add `Location` and `location` to `dlint`'s globals.
+// TODO(nayeemrmn): Add `Location`, `WorkerLocation` and `location` to `dlint`'s
+// globals.
```

Comment:
Done in https://github.com/denoland/deno_lint/pull/586

Reply:
@lucacasonato So is this PR blocked by a dlint release and `third_party` update?

Reply:
No, but we should release new version of `deno_lint` crate before `1.7.0`, I'll look into that

Reply:
Let's wait on https://github.com/denoland/deno/pull/9127 then the TODOs can be removed

Reply:
We also need to update `dlint` in `third_party`.

Reply:
Got you covered in https://github.com/denoland/deno/pull/9134